BurnOutSharp/ProtectionType/Macrovision.cs

```diff
@@ -23,12 +23,44 @@ public string CheckPortableExecutable(string file, PortableExecutable pex, bool
             if (sections == null)
                 return null;
 
-            // TODO: Add all common Macrovision PE checks here
+            // Check for generic indications of Macrovision protections first.
 
-            // Example: SafeDisc invocation
-            //string safeDisc = SafeDiscCheckPortableExecutable(file, pex, includeDebug);
-            //if (!string.IsNullOrWhiteSpace(safeDisc))
-            //    return safeDisc;
+            // Get the .text section, if it exists
+            string match = CheckSectionForProtection(file, includeDebug, pex, ".text");
+            if (!string.IsNullOrWhiteSpace(match))
+                return match;
+
+            // Get the .txt2 section, if it exists
+            match = CheckSectionForProtection(file, includeDebug, pex, ".txt2");
+            if (!string.IsNullOrWhiteSpace(match))
+                return match;
+
+            // Get the CODE section, if it exists
+            match = CheckSectionForProtection(file, includeDebug, pex, "CODE");
+            if (!string.IsNullOrWhiteSpace(match))
+                return match;
+
+            // Get the .data section, if it exists
+            match = CheckSectionForProtection(file, includeDebug, pex, ".data");
+            if (!string.IsNullOrWhiteSpace(match))
+                return match;
+
+            // Check for specific indications for individual Macrovision protections.
+
+            // Run SafeCast PE checks
+            string SafeCast = SafeCastCheckPortableExecutable(file, pex, includeDebug);
```

Comment:
For this and following, I will prefer `camelCase` variable names instead of `PascalCase` names, just so there's no confusion later.

Reply:
Would it be OK to keep PascalCase if I make the variable names more descriptive?